docs/howto/mbed-tools-howto.md

```diff
@@ -227,6 +227,14 @@ We can use a single command which will configure (set up your target and toolcha
     mbed-tools build
     ```
 
+## List library dependencies of an application
+
+All library dependencies can be listed with their URL and Git references as follows:
+
+```
+mbed-tools libs source
```

Comment:
Remove "source" here, as it isn't needed and won't be the most common way to invoke the program.

Works fine without source, as shown below:

```
% mbed-tools libs 
This program has the following library dependencies: 

Library Name    Repository URL                       Git reference
--------------  -----------------------------------  ----------------------------------------
mbed-os         https://github.com/ARMmbed/mbed-os/  890f0562dc2efb7cf76a5f010b535c2b94bce849
```

Reply:
The reason for mentioning the `source`, you could just mention the path if you are in a different directory.

Example:
```
$ mbedtools libs mbed-os-example-blinky/
This program has the following library dependencies:

Library Name    Repository URL                       Git reference
--------------  -----------------------------------  ----------------------------------------
mbed-os         https://github.com/ARMmbed/mbed-os/  0db72d0cf26539016efbe38f80d6f2cb7a3d4414
```

Reply:
Yes, but that's rare. The howto shouldn't be complete documentation. `--help` is for that. The howto should be the simplest, most-common, least confusing path.

Reply:
Fixed